### ISSUE TITLE
Change windowed mode zooming default settings

### ISSUE THREAD

**Issue by XCongoAlpha** [sylikc/jpegview#11]
### Current Default Settings: 
DefaultWindowRect=image
When you zoom in an image while in windowed mode, the window will be resized and move around like crazy.
### Proposed Default Settings: 
DefaultWindowRect=auto
When you zoom in an image while in windowed mode, the window will keep the current window size just like how every other image viewer does it. Just discovered this settings (after using JPEGView for over a year) and why isn't it the default settings? It is normally how other image viewers react when zooming in an image when in windowed mode.

This doesn't annoy me because I always backup my JPEGView.ini file but **new users who doesn't fiddle around the settings will not find this. And if they discover that zooming in windowed mode is "bugged", JPEGView will be immediately uninstalled.**

Cant attach a gif/video cause I'm too lazy to install another program just for this. Try it out and see the difference.

**Comment by sylikc** [comment]
I guess this was the way it was set before, but if it doesn't make sense, I could change it.  I don't usually see this behavior though, are you zooming into odd sized images?

**Comment by XCongoAlpha** [comment]
Happens in all image sizes

https://user-images.githubusercontent.com/74475734/146629745-7ae17ed4-f8c4-4788-9bfb-d3c11e96903e.mp4

.

**Comment by GlassGruber** [comment]
@XCongoAlpha I think this is what you are actually looking for, you have to uncheck this setting: 
`Zoom > Fit window to image` 

I came to ask basically the same thing as you but for this specific option. 
AFAICT there is no setting in the INI file for this, and when the program is closed the option is set back to active.

![image](https://user-images.githubusercontent.com/1172428/181262193-61ddeeaf-eeef-49ec-b2be-45cf21609db6.png)

Thank you @sylikc for keeping this software alive!

**Comment by JRHille** [comment]
@sylikc 
I was looking for this function in the ini file but as @GlassGruber said apparently it doesn't exist.

Without this function it is unusable for me, it is very bad that the window keeps changing size when advancing the images.

A setting to make the window sizes fixed would be nice.

**Comment by sylikc** [comment]
@JRHille  The window doesn't change if you set the setting not to change.  It's not the default setting, but you can set it in the INI

Did you set `DefaultWindowRect=auto`?

**Comment by GlassGruber** [comment]
> Without this function it is unusable for me, it is very bad that the window keeps changing size when advancing the images.

This is a bit OT but there are many ways to suggest or even ask for improvements, may I suggest to be a little more accommodating?  I'm sure you didn't mean to sound harsh.

**Comment by JRHille** [comment]
yes, I tried to make it auto and also sticky, but the window size is not fixed. I would like it to stay as when the ctrl+F11 option is unchecked.
![notepad++_O0Ofpgy3pv](https://user-images.githubusercontent.com/114708288/223194460-e3c1911f-cafa-44de-b7c5-3c6d898503d2.gif)

**Comment by JRHille** [comment]
> This is a bit OT but there are many ways to suggest or even ask for improvements, may I suggest to be a little more accommodating? I'm sure you didn't mean to sound harsh.

I apologize, but I didn't mean to be rude. I've been looking for a viewer for a long time, I've tested many and this one seems to be the best so far. But it would be nice to have an option to leave the window fixed as mentioned by the op

**Comment by sylikc** [comment]
Thanks.  So... I'm actually still very lost... Are you two saying that the CTRL+F11 feature is resetting when restarting JPEGView?

**Comment by sylikc** [comment]
@JRHille can you attach your current INI file?  I am using `auto` option myself and the window size doesn't change with image

**Comment by JRHille** [comment]
> Are you two saying that the CTRL+F11 feature is resetting when restarting JPEGView?

Exactly, the option is checked again when restarting the program.

I changed it to **auto** in both INI's, in the main JPGview folder and in %APPDATA% but it doesn't seem to have any effect, I don't know if I'm doing something wrong.

**Comment by sylikc** [comment]
Some of these defaults will are being considered in #140 but the way the user INI overrides the global INI will be addressed with suggestions in #157 

I think it's safe to mark this as a closed issue though the defaults will be changed at some point.  Or at least an "alternate" ini while retaining legacy INI ... lots of it still being designed

**Comment by JRHille** [comment]
ok, thanks a lot for your help and for keeping the program up to date.

**Comment by pTinosq** [comment]
Not to be reviving a dead issue but I've recently incorporated JPEGView into my workflow and this issue is still really quite annoying.

I have found that the workaround of `ShowFullScreen=false` and `DefaultWindowRect=auto` does (for some reason??) set the Fit Window To Image default to off which is great. The problem now is that I don't really want `DefaultWindowRect=auto`. I find `DefaultWindowRect=image` to make much more sense. If you set the setting to `image` then it just switches the fit window to image default BACK TO ON.

Not sure why this is happening but it would be amazing to see a fix for this. This is **_almost_** the perfect image viewer IMO.
